docs/developer/extending/webhooks/subscription-webhook-payloads.mdx

```diff
@@ -36,6 +36,63 @@ subscription {
 }
 ```
 
+### Filterable subscription query
```

Comment:
Maybe we could add more user-friendly or marketable name like "Filtering webhooks"?